js_basics_functions/task2.js

```diff
@@ -0,0 +1,5 @@
+const isAdult = age => age >= 18;
+const message = isAdult => `Person is ${isAdult ? '' : 'not '}adult`;
```

Comment:
I must admit this is very original solution. I like it.What I would improve

1. `message` is a function - so name it accordingly - `getMessage` as an option
2. It makes me a bit confused to see `isAdult` as a function name and as function parameter name in the same time. I would rename `isAdult` function to `getIsAdult` or something like that